src/cl/cl_node_ready_conditions.star

```diff
@@ -0,0 +1,15 @@
+def get_ready_conditions(port_id):
+    recipe = GetHttpRequestRecipe(
+        endpoint = "/eth/v1/node/health",
+        port_id = port_id
+    )
+
+    ready_conditions = ReadyCondition(
+        recipe = recipe,
+        field = "code",
+        assertion = "IN",
+        target_value = [200, 206, 503],
```

Comment:
Did you mean to leave `503` in here? If so why would that be considered a valid "ready code"?